lib/ansible/modules/cloud/amazon/sqs_queue.py

```diff
@@ -131,6 +196,19 @@
       maxReceiveCount: 5
       deadLetterTargetArn: arn:aws:sqs:eu-west-1:123456789012:my-dead-queue
 
+# Drop redrive policy
+- sqs_queue:
+    name: my-queue
+    region: ap-southeast-2
+    redrive_policy: {}
+
+# Create FIFO queue
+- sqs_queue:
+    name: fifo-queue
+    region: ap-southeast-2
+    queue_type: fifo
+    content_based_deduplication: yes
+
```

Comment:
Could you please add examples to cover the other new options; KMS, key reuse, tags, just for completeness.